package.json

```diff
@@ -18,16 +18,14 @@
   },
   "homepage": "https://github.com/EventMobi/eslint-config-eventmobi#readme",
   "dependencies": {
-    "eslint-config-airbnb": "14.0.0",
-    "eslint-config-airbnb-base": "11.0.1"
+    "eslint": "3.18.0",
+    "eslint-config-airbnb": "14.1.0",
+    "eslint-config-airbnb-base": "11.1.2",
+    "eslint-plugin-import": "2.2.0",
```

Comment:
I believe this is the reason for putting plugins in `peerDependencies`: https://github.com/eslint/eslint/issues/3458

Since ESLint can only load plugin rules from the topmost node_modules directory in the dependent project, the dependent project _must_ list all ESLint plugins in its package.json. e.g. it cannot load a plugin from `node_modules/eslint-config-eventmobi/node_modules/eslint-plugin-import`. This is mitigated by npm 3's flattening if both eslint-config-eventmobi and the dependent project specify the same version of eslint-plugin-import, but if the versions differ or if the user is using npm 2, you'll end up with the nested structure above.